src/campaign.cairo

```diff
@@ -284,11 +285,10 @@ mod TokengiverCampaign {
     #[generate_trait]
     impl InternalImpl of InternalTrait {
         fn deploy_token_giver_nft(
-            ref self: ContractState, token_giver_nft_class_hash: ClassHash, campaign_id: u256
+            ref self: ContractState, token_giver_nft_class_hash: ClassHash, admin: ContractAddress
```

Comment:
admin signifies the owner of the campaign right, if this is so, then 
it's fine, 

since each time someone denote to a campaign, we want to mint an NFT for the person.

Reply:
> admin signifies the owner of the campaign right, if this is so, then it's fine,
> 
> since each time someone denote to a campaign, we want to mint an NFT for the person.

Yes based on the constructor in `TokenGiverNFT` contract at `src/token_giver_nft.cairo`.